### DISCUSSION TITLE
Reproducibility of LAMMPS run with DP potential

### DISCUSSION THREAD

**Discussion by hl419** [deepmodeling/deepmd-kit#3269] [Q&A]
Hello,

I’m currently attempting to replicate an NVT simulation. I’ve set the seed for the initial velocity and confirmed that the initial velocity is consistent. I am also using the same machine and same version to run the simulation (single processor). However, I’ve noticed that up to a certain time-step, the positions and velocities start to deviate. I checked the previous cases #1656 and #2270, and it seems this issue came from truncation error. I wonder if there is a way to improve the precision to avoid this from happening? I would like to get a deterministic simulation that I can reproduce with exactly the same result using the same inputs. Thanks!

**Comment by wanghan-iapcm** [comment]
It is not expected to reproduce long MD trajectories due to the chaotic nature of many-body dynamical systems.
